msdsl/expr/table.py

```diff
@@ -133,8 +133,8 @@ def get_width(cls, val):
 class RealTable(Table):
     def __init__(self, vals, width=18, exp=None, name='real_table', dir='.',
                  real_type=None, rec_fn_sig_width=None, rec_fn_exp_width=None):
-        # calculate the range of values
-        range_ = max([abs(val) for val in vals])
+        # calculate the range of values (slightly overestimated to prevent range violations due to rounding)
+        range_ = max([abs(val) for val in vals])*1.01
```

Comment:
As with the previous change, it's important to make sure there is a bit of margin in the real-number formatting, and since svreal doesn't know about arrays/tables, msdsl has to handle this case.